src/pages/Designer/Designer.js

```diff
@@ -0,0 +1,253 @@
+  BarChart,
+} from 'grommet-icons';
+import { grommetToolsTheme } from '../../theme.js';
+import AppHeader from '../../components/AppHeader.js';
+import IconCard from '../../components/IconCard.js';
+
+const Designer = () => (
+  <Box>
+    <AppHeader />
+    <Main>
+      <Box margin={{ top: 'xlarge' }}>
+        <Stack>
+          <Box>
+            <Box alignSelf="center">
+              <Text
+                margin={{
+                  left: 'large',
+                }}
+              >
+                Grommet Tools
+              </Text>
+              <Heading
+                size="xlarge"
+                margin={{
+                  top: 'none',
+                  bottom: 'medium',
+                  left: 'large',
+                }}
+              >
+                Designer
+              </Heading>
+            </Box>
+            <Box height="gradient1" background="gradient"></Box>
+            <Box alignSelf="center" margin={{ left: 'xlarge' }} width="large">
+              <Paragraph
+                margin={{
+                  left: 'xlarge',
+                }}
+                size="xxlarge"
+                color="darkGrey"
+              >
+                Grommet Designer is a tool used build expiriences with Grommet
+                components; then publish and share your ideas with a simple
+                wysiwg interface
+              </Paragraph>
+              <Box
+                round="medium"
+                background="buttonYellow"
+                onClick={() => {}}
+                width="small"
+                height="xxsmall"
+                justify="center"
+                margin={{ left: 'xlarge' }}
+              >
+                <Text weight="bold" alignSelf="center" color="black">
+                  Open Designer
+                </Text>
+              </Box>
+            </Box>
+          </Box>
+          <IconCard
+            size="medium"
+            icon={<ClearOption size="xxlarge" color="DesignerYellow" />}
+            margin={{ left: 'xlarge' }}
+          />
+        </Stack>
+      </Box>
+      <Box
+        direction="column"
+        gap="medium"
+        margin={{ left: 'xlarge', bottom: 'xlarge' }}
+      >
+        <Box direction="row" margin={{ left: 'xlarge' }} gap="xlarge">
+          <Camera size="large" color="ImagerOrange" />
+          <Configure size="large" color="ThemerOrange" />
+        </Box>
+        <Box margin={{ left: 'xlarge' }} direction="row">
+          <Box width="xxsmall"></Box>
+          <CloudUpload size="large" color="PublisherPink" />
+        </Box>
+        <Box margin={{ left: 'xlarge' }} direction="row">
+          <Box width="xsmall"></Box>
+          <Selection size="large" color="SlidesBlue" />
+        </Box>
+        <Box margin={{ left: 'xlarge' }}>
+          <BarChart size="large" color="TabularGreen" />
+        </Box>
+      </Box>
+
+      <Stack>
+        <Box
+          background="YellowBackground"
+          height="large"
```

Comment:
Do you really need the height? could you try to make the content fit this height instead?

Reply:
> Why do we need so many designer.svgs?

They are all of the various screenshots that are on the page. I think there are 6 screenshots plus the gremlin holding the laptop

Reply:
I used the height because the cards are overhanging the bottom of the box